chapter-2/chapter2.md

```diff
@@ -0,0 +1,20 @@
+# Overview of Java
+### *Priciples of OOP*
+
+1. Encapsulation: Mechanism that binds code and data together to keep safe from outside interference an misuse.
```

Comment:
rename the file name from chapter2.md to README.md

Reply:
Done vaiya.